lib/flow/status.js

```diff
@@ -139,6 +143,12 @@ module.exports = {
     id: 'rejected',
     commentRequired: true
   },
+  recovered: {
+    id: 'recovered',
+    commentRequired: false,
```

Comment:
Should we enforce an explanation, or do we think it's not necessary?

Reply:
it is a button style form with a js.confirm dialog, I think that's fine as it's only going to realistically be done by Martin